nifi-system-tests/nifi-system-test-suite/src/test/resources/simplelogger.properties

```diff
@@ -0,0 +1,2 @@
+org.slf4j.simpleLogger.showDateTime=true
```

Comment:
Added this in order to generate timestamps for system-test-suite log messages.  The timestamps are very useful for correlating test case trace to the corresponding NiFi trace.